.travis.yml

```diff
@@ -6,7 +6,7 @@ language: objective-c
 env:
   matrix:
     
-    - CONDA_PY=35
+    - CONDA_PY=352
```

Comment:
This same issue occurred when the feedstock was initially rendered, see #11 and #20 which fixed the issue.